src/Runner.Common/Terminal.cs

```diff
@@ -27,14 +28,15 @@ public interface ITerminal : IRunnerService, IDisposable
 
     public sealed class Terminal : RunnerService, ITerminal
     {
-        public bool Silent { get; set; }
```

Comment:
Repurposing the unused `Silent` prop as `PrintLogToStdout` to avoid double printing

Reply:
`Silent` was used by testing, I think. do we want to keep that?